src/demo/components/TextboxLayout.css

```diff
@@ -31,5 +31,20 @@
     pointer-events: none;
     font-weight: 500;
   }
- 
-  
\ No newline at end of file
+@media only screen and (min-width: 325px) and (max-width: 767px) {
+  .input-container{
+    padding: 0.7rem 1rem;
+    margin-bottom: 1rem;
+  }
+  .transparent-textbox{
+    padding: 1rem 1.4rem 1.4rem 1.4rem;
+    height: 40%
+  }
+  .textbox-label {
+    position: absolute;
+    top: -1.1rem;
```

Comment:
negative absolute positions can be tricky if they dont know they have the padding space.